keyboards/gboards/gergoplex/config.h

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Copy and worked on with love from the EZ team
+
+#pragma once
+#include "config_common.h"
+
+#define BOOTMAGIC_LITE_ROW 9
+#define BOOTMAGIC_LITE_COLUMN 2
+
+#define VERBOSE
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x0007
+#define PRODUCT_ID 0x0002
+#define DEVICE_VER 0x0001
+#define MANUFACTURER g Heavy Industries
+#define PRODUCT GergoPlex
+#define DESCRIPTION More ErgoGergo
+
+/* key matrix size */
+#define MATRIX_ROWS 10
+#define MATRIX_ROWS_PER_SIDE (MATRIX_ROWS / 2)
+#define MATRIX_COLS 4
+
+#define MOUSEKEY_INTERVAL 20
+#define MOUSEKEY_DELAY 0
+#define MOUSEKEY_TIME_TO_MAX 60
+#define MOUSEKEY_MAX_SPEED 7
+#define MOUSEKEY_WHEEL_DELAY 0
+#define TAPPING_TOGGLE 1
+#define TAPPING_TERM 200
+#define IGNORE_MOD_TAP_INTERRUPT
```

Comment:
Just out of curiousity, are these type of settings (`IGNORE_MOD_TAP_INTERRUPT`, `TAPPING_TERM`, `COMBO_TERM` etc.) appropriate to define at a keyboard level? When I try to override them with my own `config.h` I get an error due to duplicate definitions. Is there a proper way of doing that? (Or say, disabling `IGNORE_MOD_TAP_INTERRUPT`?)

Reply:
They can be.  

As for un-setting them or changing them, you'd need to `#undef` them first

Reply:
Thanks, `#undef` was the missing piece for me.

Reply:
> They can be.
> 
> As for un-setting them or changing them, you'd need to `#undef` them first

Should there be a limit on what is set at the keyboard level?  Otherwise every user needs to undef every existing and future QMK setting in order to get predictable behaviour at the keymap level.